label_studio_converter/utils.py

```diff
@@ -55,7 +67,7 @@ def create_tokens_and_tags(text, spans):
         tokens, tags = [], []
         for token, token_start in tokens_and_idx:
             tokens.append(token)
-            token_end = token_start + len(token) #"- 1" - This substraction is wrong. token already uses the index E.g. "Hello" is 0-4
+            token_end = token_start + len(token) - 1#"- 1" - This substraction is wrong. token already uses the index E.g. "Hello" is 0-4
```

Comment:
If "this subtraction is wrong", why are we doing a subtraction here? Can we clarify or remove this comment?

Reply:
It was wrong before when **tokenize** method was used to separate tokens.